src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -70,6 +70,7 @@
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.32" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="5.0.0-beta.1" />
     <PackageReference Include="Microsoft.Azure.WebSites.DataProtection" Version="2.1.91-alpha" />
+    <PackageReference Include="Microsoft.Security.Utilities" Version="1.1.0" />
```

Comment:
>Utilities [](http://example.com/codeflow?start=50&length=9)

This package contains key generation logic (and tests) that have been integrated in several other Azure resource providers. Note there is a pending change in this package possibly relevant to your RP (to produce URL friendly base64-encoded keys) that we could consider to update this PR.

Reply:
What nuget feed is this on? Looks like on nuget.org the name is just reserved https://www.nuget.org/packages/Microsoft.Security.Utilities/

Reply:
Can you see this version? https://www.nuget.org/packages/Microsoft.Security.Utilities/1.2.0

Reply:
We just published v1.3 of this package, in order to provide a helper that generates a URL safe token without eliding the equal sign padding. We completed some offline analysis that suggests this will improve the accuracy of the preliminary regex match.